src/reducers/conversationReducer.js

```diff
@@ -14,30 +14,34 @@ import {
   USER_UPDATED_EVENT,
   MARK_AS_READ_EVENT,
   DELETE_MESSAGE_FOR_EVERYONE_EVENT,
-  DELETE_MESSAGE_EVENT
+  DELETE_MESSAGE_EVENT,
+  CONVERSATION_BAN_LIST_SUCCESS,
+  BAN_CONVERSATION_USERS_SUCCESS,
+  UNBAN_CONVERSATION_USERS_SUCCESS,
```

Comment:
@pardeep-kumar-bs we should move this to `messageReducer` as this reducer is for conversation list.

Reply:
Done.